packages/OmegaPrint-Core.package/OPPrinter.class/instance/BlockLiteralNormal.with.and.and.and..st

```diff
@@ -1,17 +1,15 @@
 actions
 BlockLiteralNormal: aNode with: aTerminal and: blockArguments and: executableCode and: anotherTerminal
 
-	| result resultArguments resultCode space |
-	result := '['.
+	| result space |
```

Comment:
space in separator umbenennen -> Konsistenz KeywordMessageSend

Reply:
ah, sehr guter Punkt, habe ich bestimmt auch anderswo falsch gemacht